lib/board.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class Board
+  attr_reader :board, :avail_squares
+  attr_writer :board, :avail_squares
+
+  def initialize
+    @board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+    @avail_squares = board.dup
+  end
+
+  def display_board
+    puts ' '
+    puts " #{board[0]}  |  #{board[1]}  |  #{board[2]}"
+    puts '----|-----|-----'
+    puts " #{board[3]}  |  #{board[4]}  |  #{board[5]}"
+    puts '----|-----|-----'
+    puts " #{board[6]}  |  #{board[7]}  |  #{board[8]}"
+    puts ' '
+  end
```

Comment:
This method should be moved to `main.rb`

Reply:
Hello, we removed the puts to comply, left the method in board tho, since I believe this method should be in the board class. thanks for the review , hope it is alright now :)